setup.py

```diff
@@ -5,11 +5,17 @@
 from io import open
 import re
 try:
-    from azure_bdist_wheel import cmdclass
+    import azure
+    try:
+        ver = azure.__version__
```

Comment:
This try except logic was to test "azure" package as it was 3.5 years ago. I think it's safe for you to not introduce it and keep you file simple.

Reply:
Can you confirm the changes I pushed? We don't need to import azure package, right?

Reply:
Yes we don't, changes looks good to me